Binary files DTW_SO_Test/_DTW.so and DTW_SO_Testf/_DTW.so differ
diff -ruN DTW_SO_Test/Makefile DTW_SO_Testf/Makefile
--- DTW_SO_Test/Makefile	2007-11-09 17:38:15.000000000 +0100
+++ DTW_SO_Testf/Makefile	2008-01-26 13:23:40.000000000 +0100
@@ -1,4 +1,4 @@
 make:
-	gcc -c main.c
+	gcc -fPIC -c main.c
 	ld -shared main.o -o _DTW.so
 	rm main.o
